diagnostic.md

```diff
@@ -9,6 +9,12 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+Numbers: 2, 3, 4
+Booleans: true, false
+Strings: "Hi, my name is Josh"
+null: value of 0
+undefined: nothing has been defined
+
```

Comment:
Correct on primitive types, just note that null's only example available is a literal `null` value.